lib/codecs/src/encoding/format/raw_message.rs

```diff
@@ -48,7 +48,7 @@ impl Encoder<Event> for RawMessageSerializer {
                 .get_by_meaning(message_key)
                 .or_else(|| log.get(message_key)) // backward compatibility
                 .map(|value| value.coerce_to_bytes()),
-            Event::Metric(_) => None,
+            Event::Metric(metric) => Some(metric.to_string().into()),
```

Comment:
Is this prometheus exposition format? Or something else?

Reply:
Something similar I suppose:

https://github.com/vectordotdev/vector/blob/1c87d918b53b04c88274e12027519a1c27a55741/lib/vector-core/src/event/metric/mod.rs#L361-L382